src/app/collective-rewards/rewards/backers/context/BackerRewardsContext.tsx

```diff
@@ -92,9 +92,15 @@ const getEarnedAddresses = (rewards: Record<Address, bigint>) =>
 export const BackerRewardsContextProvider: FC<BackerRewardsProviderProps> = ({
   children,
   backer,
-  gauges,
   tokens: { rif, rbtc },
 }) => {
+  const { builders, isLoading: buildersLoading, error: buildersError } = useBuilderContext()
+  const { activeBuilders, gauges } = useMemo(() => {
```

Comment:
it looks like we're not using `activatedBuilders`

Reply:
yeah let me remove it, since we just need the gauges